test/data/repositories/rhel-84.json

```diff
@@ -0,0 +1,109 @@
+      "check_gpg": true,
+      "image_type_tags": [
+        "gce"
+      ]
+    },
+    {
+      "name": "eus",
+      "baseurl": "https://cdn.redhat.com/content/eus/rhel8/8.4/x86_64/appstream/os",
```

Comment:
You should define the EUS repo also for BaseOS and also do it for all architectures.